File: fix-quotes.js
const fs = require('fs');

const filepath = process.argv[2] || 'getting-started/get-started-with-mainwp.mdx';
let content = fs.readFileSync(filepath, 'utf-8');

// Replace curly double quotes with straight single quotes
content = content.replace(/\u201c/g, "'");
content = content.replace(/\u201d/g, "'");

fs.writeFileSync(filepath, content, 'utf-8');
console.log('Fixed curly quotes in', filepath);

File: fix_quotes.py
#!/usr/bin/env python3
import sys

filepath = sys.argv[1]

with open(filepath, 'r', encoding='utf-8') as f:
    content = f.read()

# Replace curly quotes with straight single quotes
content = content.replace('\u201c', "'")  # Left double quotation mark
content = content.replace('\u201d', "'")  # Right double quotation mark

with open(filepath, 'w', encoding='utf-8') as f:
    f.write(content)

print(f'Fixed curly quotes in {filepath}')

